x-pack/plugins/fleet/server/services/security/fleet_router.ts

```diff
@@ -82,6 +84,31 @@ export function makeRouterWithFleetAuthz<TContext extends FleetRequestHandlerCon
       logger.info(`User does not have required fleet authz to access path: ${request.route.path}`);
       return response.forbidden();
     }
+
+    // No need to check for EPM requests
+    if (request.route.path.includes('api/fleet')) {
```

Comment:
This is what a lock will looks like, it will introduce an extra request to retrieve a saved object on every API request

Reply:
I guess now that we lock at the client level we can remove this HTTP layer lock? Since all operations go through the client, right?